can-bus/json/DCM/DCM_tx.json

```diff
@@ -144,6 +130,9 @@
             "StackWatermarkAboveThresholdTask1Hz": {
                 "bits": 1
             },
+            "StackWatermarkAboveThresholdTask100Hz": {
+                "bits": 1
+            },
```

Comment:
This was actually intentional. I noticed a number of boards did not have a StackWatermarkAboveThresholdTask100Hz so I added them to any board I was reading from the DIM. I'm assuming the 100Hz task was added after the watermark warnings were created and they never got updated